API Explorer announcement (#315)
bump-sh/docs: src/_product-updates/2025-01-09-api-explorer.md

---
title: Introducing the API Explorer
tags: [New]
image: /images/changelog/api-explorer.png
---
![api-explorer.png](/images/changelog/api-explorer.png)

> We are happy to announce the launch of our API Explorer, our client’s most requested feature.

Reviewer: ```suggestion
We are happy to announce the launch of our API Explorer, our clients' most requested feature.
```

Sending a first request is the last mile of an API discovery. As setting up an API client or using cURL in a CLI can be an endeavour for API users, we made things easier, by integrating the API Explorer into your documentation portal. 

> With the API Explorer, enable your users to go from API discovery to API usage, in one place.

Reviewer: ```suggestion
With the API Explorer, enable your users to go from API discovery to API usage, in single place.
```

## Send requests, right next to the API documentation
It can be difficult for API consumers to understand the complete behavior of an API, that’s one of the reasons we make documentation. To ease the API’s discoverability in the Explorer, the request form is automatically generated with the right field types, based on the OpenAPI definition. No more Lorem Ipsum inside a date picker.

Readability is key in a context of discovery. A dedicated view for the API Explorer next to the documentation means focusing on what’s important: the customer’s request, and the API’s response, while keeping the documentation just one click away. 

![api-explorer-request.png](/images/changelog/api-explorer-request.png)

## Deep support of OpenAPI

> Your [API definition file is a contract](https://bump.sh/blog/how-openapi-ensures-reliable-api-communication) between you and your users. That's why we went all-in on supporting the OpenAPI specification in our API Explorer. From simple properties such as strings and booleans, to more complex ones such as arrays of objects or oneOf/anyOf, try any sort of request, based on any API design.

Reviewer: ```suggestion
Your [API definition file is a contract](https://bump.sh/blog/how-openapi-ensures-reliable-api-communication) between you and your users. That's why we went all-in on supporting the OpenAPI specification in our API Explorer. From simple properties such as strings and booleans, to more complex ones such as arrays or oneOf/anyOf, try any sort of request, based on any API design.
```

Author: I think in this case it's nice to talk about a complex array, such as an array of objects. 
But we indeed support all kind of arrays.
